mmv1/products/vertexai/api.yaml

```diff
@@ -997,3 +999,208 @@ objects:
           The update method to use with this Index. The value must be the followings. If not set, BATCH_UPDATE will be used by default.
           * BATCH_UPDATE: user can call indexes.patch with files on Cloud Storage of datapoints to update.
           * STREAM_UPDATE: user can call indexes.upsertDatapoints/DeleteDatapoints to update the Index and the updates will be applied in corresponding DeployedIndexes in nearly real-time.
+
+# Vertex AI Models
+  - !ruby/object:Api::Resource
+    name: Model
+    base_url: projects/{{project}}/locations/{{location}}/models
+    create_url: projects/{{project}}/locations/{{location}}/models:upload
+    self_link: 'projects/{{project}}/locations/{{location}}/models/{{name}}'
+    update_verb: :PATCH
```

Comment:
Can you add an update test for the resource? We'll want to update all updatable fields in it.